src/test/scala/tech/sourced/gemini/BaseDBSpec.scala

```diff
@@ -0,0 +1,55 @@
+package tech.sourced.gemini
+
+import com.datastax.driver.core.{Cluster, Session}
+import org.scalatest.{BeforeAndAfterAll, Suite, Tag}
+
+case class hashtableItem(hashtable: Int, v: String, sha1: String)
```

Comment:
The Go convention of lower-case for private stuff does not apply to Scala ;-) https://docs.scala-lang.org/style/naming-conventions.html